### PR TITLE
Site alert

### PR DESCRIPTION
**Pull request by liamjohnston**
Needs a bunch of BED work, but ready for FED review.

Screenshot 2024-06-18 at 4 38 29 PM

Screenshot 2024-06-18 at 4 38 49 PM

### CODE REVIEW

File: cdhweb/static_src/global/components/site-alert.scss [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  );
+
+  line-height: 1.5;
+  border-block-end: 1px solid var(--color-brand-110);
+}
+
+.alert-banner--informational {
+  --alert-accent-color: var(--color-brand-100);
+}
+.alert-banner--warning {
```
Comment (cameralibre) [inline cdhweb/static_src/global/components/site-alert.scss]: You could put these selectors together with a comma to make it clear that the same happens for both.  Or set this variable on `.alert-banner` and only override it with `--emergency`

**Review by cameralibre — APPROVED** [review]
FED good ✅

File: cdhweb/pages/templatetags/core_tags.py [hunk 2]
```diff
@@ -176,3 +176,16 @@ def secondary_navigation():
...
+
+@register.inclusion_tag("includes/site_alert.html", takes_context=True)
+def site_alerts(context):
+    now = timezone.now()
+    site_alerts = (
+        SiteAlert.objects.all()
+        .exclude(display_from__gt=now)
+        .exclude(display_until__lt=now)
+    )
+    print(site_alerts)
```
Comment (haydngreatnews) [inline cdhweb/pages/templatetags/core_tags.py]: Debugging

**Review by haydngreatnews — APPROVED** [review]

File: cdhweb/pages/templatetags/core_tags.py (hunk 2 above)
Comment (liamjohnston) [reply cdhweb/pages/templatetags/core_tags.py]: fixed

**Pull request merged**